molgenis-core-ui/src/main/javascript/modules/react-components/AggregateTable.js

```diff
@@ -39,7 +41,7 @@
 		},
 		render: function() {
 			if(this.state.data === null) {
-				return molgenis.ui.Spinner(); // entity not available yet
+				return Spinner(); // entity not available yet
```

Comment:
We return spinners like crazy. Is there not a more elegant way to return the component that you actually want to show and that that will render a spinner if not ready (I am looking for a way to reduce this code).
Moreover, we now throw many seperate spinners while ideally you would want only one, i.e., only the container throws a spinner (sorry, I am not being very clear am I?)